README.md

```diff
@@ -82,6 +86,33 @@ The [contribution guide](CONTRIBUTING.md) describes how to use the `create-rule`
 
 For more advanced command line interface (CLI) usage, refer to the [CLI guide](CLI.md).
 
+## Red Team Automation
```

Comment:
thanks for adding this!
i'm wondering if this makes sense in a separate file.
also i'm wondering if a `rta/README.md` file would make sens

Reply:
I like that Idea, I can certainly do that